shared/Cargo.toml

```diff
@@ -17,9 +17,12 @@ testing = [
   "rand_core",
 ]
 wasm-runtime = [
+  "ibc",
```

Comment:
instead of grouping these with `wasm-runtime` I think it maybe worth separating them into a new feature. Then only enable this in `apps/Cargo.toml` for `anoma_shared`

Reply:
That looks better. Thank you! I added `ibc-vp` feature.